.ci/test.jl

```diff
@@ -4,5 +4,8 @@ pkgs = [
     "KernelAbstractions",
     "CUDAKernels",
 ]
+if !(VERSION < v"1.6-")
```

Comment:
Unconditional?

Reply:
These are here because AMDGPU is 1.6-only (anything pre-1.6 is basically garbage), so you'd hit resolver errors trying to develop or test it.

Reply:
Yeah I thought KA 0.6 was 1.6 only already